src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -81,21 +91,57 @@ public Generator(int population) {
    * 
    * @param population Target population size
    * @param seed Seed used for randomness
+ * @throws IOException 
    */
-  public Generator(int population, long seed) {
+  public Generator(int population, long seed) throws IOException {
     init(population, seed, DEFAULT_STATE, null);
   }
 
   /**
    * Create a Generator, with the given options.
    * @param o Desired configuration options
+ * @throws IOException 
    */
-  public Generator(GeneratorOptions o) {
+  public Generator(GeneratorOptions o) throws IOException {
     String state = o.state == null ? DEFAULT_STATE : o.state;
     init(o.population, o.seed, state, o.city);
   }
+  
+  @SuppressWarnings("rawtypes")
+  private static Map relationships() {
```

Comment:
Don't think this was intended to be here - it's called relationships() but references the cdc_growth_charts file